flake8_isort.py

```diff
@@ -25,6 +27,15 @@ class Flake8Isort(object):
     no_config_msg = (
         'I002 no configuration found (.isort.cfg or [isort] on setup.cfg)'
     )
+    isort_unsorted = (
+        'I003 isort found an import in the wrong position'
```

Comment:
if ``I001`` is no longer used you can move up all the error codes, unless you plan to add a config switch, which I wouldn't really